test/visibility/private/moduleSymbols/accessPrivateUsedSibling2.good

```diff
@@ -0,0 +1,3 @@
+accessPrivateUsedSibling2.chpl:18: In function 'main':
```

Comment:
I'd be tempted to make this and the variable version semantic futures that reference the issue Michael opened

Reply:
I don't see a big difference between a working test with an open (active) issue and a future with no well-defined `.good` file.  Either one is going to announce when the behavior changes and the future isn't going to start passing at some point in the future.  If someone else wanted to file a future advocating for different output, that would be OK with me, but
for simplicity / to save time, I'd prefer to leave them as-is (also because I don't think the current behavior is obviously wrong).